samples/sensor/mcp9600/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  name: MCP9600 Temperature Sensor
+tests:
+  sample.sensor.mcp9600:
+    harness: sensor
+    tags: sensors
+    harness_config:
+        type: one_line
+        regex:
+            - "ttemp: (.*)"
+        fixture: fixture_i2c_mcp9600
+    depends_on: i2c
+    filter: dt_compat_enabled("microchip,mcp9600")
```

Comment:
I don't think this will pick up the boards you added overlays for, so please replace this with a platform_allow list

Reply:
Switched to the MCP9808 sample.yaml, hope that is acceptable as Peter mentioned not sure what boards/chips are supported.